### PR TITLE
handle differnt api url depending on the type of operation

### PR DESCRIPTION
**Pull request by pruan-rht**
addresses comment in issue https://github.com/openshift/verification-tests/issues/118

### CODE REVIEW

File: lib/rest_kubernetes.rb [hunk 1]
```diff
@@ -5,8 +5,10 @@ module Rest
     module Kubernetes
       extend Helper
 
-      def self.populate(path, base_opts, opts)
-        populate_common("/api/<api_version>", path, base_opts, opts)
+      # the api url can either be /api/<api_version> or /apis/#{type}.k8s.io/<api_version> depending on the operation
+      def self.populate(path, base_opts, opts, type: nil)
+        type_str = type ? "s/#{type}.k8s.io" : ""
```
Comment (akostadinov) [inline lib/rest_kubernetes.rb]: looks like unintended `s` needs to be removed

Comment (pruan-rht) [reply lib/rest_kubernetes.rb]: @akostadinov, no, the `s` should be there according to https://docs.okd.io/latest/rest_api/

Comment (akostadinov) [reply lib/rest_kubernetes.rb]: my bad, missed that

**Pull request merged**